code/game/objects/obj_defense.dm

```diff
@@ -284,6 +288,6 @@ GLOBAL_DATUM_INIT(acid_overlay, /mutable_appearance, mutable_appearance('icons/e
 		return TRUE
 	return FALSE
 
-//returns how much the object blocks an explosion
+///returns how much the object blocks an explosion. Currently not implemented
```

Comment:
"Currently not implemented" does not accurately describe this.

The proc is only called if `explosion_block` var is set to `EXPLOSION_BLOCK_PROC`. Doors and windows set this var and override this proc.